### DISCUSSION TITLE
Daisy UI and Static pages

### DISCUSSION THREAD

**Discussion by KenA67** [saadeghi/daisyui#747] [General]
Hello. I'm aware that I can use TailwindCSS with Static pages, all I got to do is install Tailwind then `npx tailwindcss build main.css -o output.css` and just use the `output.css` file with the static html page.

I haven't tried it yet, but I believe since DaisyUI is a plugin to TailwindCSS, after I have DaisyUI installed and then I run the `npx tailwindcss...` command, the generated `output.css` file will contain any DaisyUI css I have used, right?

**Comment by saadeghi** [comment]
Yes.
After you set up Tailwind CSS,
Install daisyUI:
```
npm i daisyui
```
Then add it to your tailwind.config.js:
```js
module.exports = {
  //...
  plugins: [require("daisyui")],
}
```
And daisyUI classes will be added to Tailwind CSS as a plugin so it works the same way Tailwind works.

**Reply by KenA67** [reply]
Yeap, thanks @saadeghi ... works perfectly ...
ps.: in case someone else is doing static pages, in the `module.exports` don't forget to add the `content` also, eg: `  content: ["./**/*.{html,js}"],`
